supply/lib/supply/client.rb

```diff
@@ -410,6 +410,23 @@ def track_version_codes(track)
       end
     end
 
+    # Get list of release names for track
```

Comment:
The addition of this method seems to make rubocop unhappy:

```
supply/lib/supply/client.rb:98:3: C: Metrics/ClassLength: Class has too many lines. [325/320]
  class Client < AbstractGoogleServiceClient ...
  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Is there a useful way to extract this? If not, feel free to override that one rule here I would say.

Reply:
Seems a little over kill to extract this method alone. The Client obviously needs some work to make this happen.
I'll have a think of a possible approach, input welcome

Reply:
As I said:

> If not, feel free to override that one rule here I would say.